tests/cross-product/sapi/scenarios/checkout/shared-checkout-scenario.js

```diff
@@ -0,0 +1,49 @@
+import { AbstractScenario } from '../../../../abstract-scenario.js';
+import { group } from 'k6';
+
+export class SharedCheckoutScenario extends AbstractScenario {
+    execute() {
+        let self = this;
+
+        group('Checkout', function () {
+            self.checkoutWith1Item();
+            self.checkoutWith70Items();
+        });
+    }
+
+    checkoutWith1Item() {
+        let self = this;
+
+        const cartUuid = self.cartHelper.haveCartWithProducts(0);
+        const response = this.checkout(cartUuid, 1, __ENV.sku);
+
+        self.assertResponseStatus(response, 201);
+    }
+
+    checkoutWith70Items() {
+        let self = this;
+
+        const cartUuid = self.cartHelper.haveCartWithProducts(0);
+        const response = this.checkout(cartUuid, 70, __ENV.sku);
+
+        self.assertResponseStatus(response, 201);
+    }
```

Comment:
We must have 2 separate tests for 1 and 70 items to be able to run them separately.

Reply:
it was suggested to combine them to 1